### DISCUSSION TITLE
求个小黑盒的签到

### DISCUSSION THREAD

**Discussion by wintsa123** [qd-today/templates#725] [Q&A]
- 名称：小黑盒
- 类型：游戏论坛
- 网址：[小黑盒](https://www.xiaoheihe.cn/home/)
- 描述：就普通签到功能

**Comment by hhhllll888** [comment]
这得抓包吧，但是现在只要打开app就自动签到了，不需要自己手动签到了，如果经常玩基本就两三百天了

**Reply by ARening** [reply]
也不想天天登录哈哈

**Reply by hhhllll888** [reply]
直接手机设置自动打开app不就行了，小黑盒自动打开就签到了啊

**Reply by ARening** [reply]
还能这样吗？我回头找找资料
